ppr-ui/src/interfaces/ppr-api-interfaces/registration-interfaces.ts

```diff
@@ -34,8 +34,35 @@ export interface FinancingStatementIF {
 // Draft interface. TODO: add change, amendment statement draft definitions when available.
 export interface DraftIF {
     type: string, // One of enum DraftTypes.
+    documentId?: string,
```

Comment:
DraftIF represents an API draft, which may be for a financing statement, a change statement, or an amendment statement. FinancingStatementIF holds all the API information for a draft financing statement. There is some duplication here, and in the RegistrationIF definition (which is not in the API). When an amendment draft is implemented it will be added to DraftIF as AmendmentStatementIF.

Reply:
I see what you're saying here, but the documentId is not coming back under a financingStatement key 
financingStatement : { documentId: "abc" }
so I guess some translation would need to happen in the ppr-api helper...

Reply:
I've reworked this now